### ISSUE TITLE
update

### ISSUE THREAD

**Issue by shahkalp09** [shahkalp09/character-embbeding#3]
sir, please check I have uploaded the char cnn code with the imbd dataset on GitHub with the requirements text file in the branch name feature_dev/kalp. Now I am working on another dataset for the same CNN model .

**Comment by knitemblazor** [comment]
good work.

**Comment by shahkalp09** [comment]
sir, completed the cnn model with the given data and using PyTorch custom data loader, I have uploaded the code-named "cnn_final"

**Comment by knitemblazor** [comment]
yes that should be fine for now. do add comments in the code per function etc try modularizing the code a bit more.

**Comment by shahkalp09** [comment]
25/5/23: Completed task 2 of making inference pipeline and Task 3 of resolving merging conflicts.

**Comment by shahkalp09** [comment]
29/5/23: Done with coding and resolving the errors for task 4

**Comment by shahkalp09** [comment]
30/5/23: Subtask 1 completed with the basic model and worked on subtask 2

**Comment by shahkalp09** [comment]
31/5/23: increasing the modularity of Subtask 1, tried subtask1 using GNN and resolving its errors, also trying to implement suggestions, and working on Subtask 2  of Task 4.

**Comment by shahkalp09** [comment]
9/5/23:sir, done with the literature review of TAPAS, Working on creating the dataset feasible for TAPAS model.
